### ISSUE TITLE
FunctionApp with netstandard2.0 dependency not working

### ISSUE THREAD

**Issue by dersia** [Azure/azure-webjobs-sdk-script#1972]
I created a simple repro: https://github.com/dersia/afnetstandard2.0test

Running this manual triggered function from the portal logs out the following:
```
2017-09-28T23:43:57.937 Function started (Id=00bb0302-3413-4de1-960b-54db655144c4)
2017-09-28T23:43:58.234 Exception while executing function: Functions.ManualTriggerCSharp1. mscorlib: Exception has been thrown by the target of an invocation. f-ManualTriggerCSharp1__-1357806294: Could not load file or assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.
2017-09-28T23:43:58.283 Function completed (Failure, Id=00bb0302-3413-4de1-960b-54db655144c4, Duration=351ms)
```

The function app is running on the CosumptionPlan in CanadaCentral and EastUS

**Comment by dersia** [comment]
I think switching to net471 could solve this issue:
https://blogs.msdn.microsoft.com/dotnet/2017/09/28/net-framework-4-7-1-runtime-and-compiler-features/

**Comment by fabiocav** [comment]
Resolved by #2042
